# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/sqlite/hotfix-autoconf.patch.nios2
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- sqlite-autoconf-3500300/autosetup/autosetup-config.guess.vanilla	2025-07-24 12:31:32.506928248 +0200
+++ sqlite-autoconf-3500300/autosetup/autosetup-config.guess	2025-07-24 12:33:49.159319798 +0200
@@ -1159,6 +1159,9 @@
 	  *)    GUESS=hppa-unknown-linux-$LIBC ;;
 	esac
 	;;
+    nios2:Linux:*:*)
+	GUESS=nios2-unknown-linux-$LIBC
+	;;
     ppc64:Linux:*:*)
 	GUESS=powerpc64-unknown-linux-$LIBC
 	;;
